std/log/logger.ts

```diff
@@ -37,86 +36,67 @@ export class LogRecord {
   }
 }
 
-export interface LoggerOptions {
-  handlers?: BaseHandler[];
-}
+const DEFAULT_LEVEL = "Debug";
```

Comment:
Shouldn't this be `LogLevelNames.Debug`?

Also the README should probably mention what the default log level is.

Reply:
Also, this changed from what was previously `INFO`. Why? Please revert.

Reply:
@vwkd actually I found that the default log level is different for different logging frameworks.
In java its `Info`, for rust and logLevel its `warn`, though they have  I just think the default should be all inclusive and if some messages should not be logged, there should be a specific log level set.



Yes, it should be a `LogLevelNames` property instead of a string.